thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/controller.js

```diff
@@ -127,6 +127,15 @@ export default Ember.Controller.extend({
   ),
 
   /**
+   * We only want to display the custom legend if the alert has a value for dimension exploration
+   * @method showGraphLegend
+   * @return {Boolean}
+   */
+  showGraphLegend: Ember.computed('alertDimension', function() {
```

Comment:
fyi: I think this should work also 
```
showGraphLegend: Ember.computed.notEmpty('alertDimension')
```
https://www.emberjs.com/api/ember/2.15/namespaces/Ember.computed/methods/bool?anchor=notEmpty

Reply:
nice - thanks.